dropshot_endpoint/src/endpoint.rs

```diff
@@ -834,6 +1044,57 @@ mod tests {
         );
     }
 
+    // These argument types are close to being invalid, but are actually valid.
```

Comment:
Is it that these are odd but valid types? or could you elaborate on this proximity to invalidity?

Reply:
Expanded the comment here to describe the changes which would make these types invalid.